core/src/main/java/org/testcontainers/utility/DockerLoggerFactory.java

```diff
@@ -17,10 +17,10 @@ public static Logger getLogger(String dockerImageName) {
             abbreviatedName = dockerImageName;
         }
 
-        if ("UTF-8".equals(System.getProperty("file.encoding"))) {
-            return LoggerFactory.getLogger("\uD83D\uDC33 [" + abbreviatedName + "]");
-        } else {
-            return LoggerFactory.getLogger("docker[" + abbreviatedName + "]");
-        }
+        return LoggerFactory.getLogger(String.format(
+            "docker.%s[%s]",
```

Comment:
I'd suggest avoiding brackets here. Some frameworks (e.g. Quarkus) give brackets a special meaning, and as a consequence, the following configuration would fail:

```properties
quarkus.log.category."docker[foo:latest]".level=ERROR
```

Of course, this would work:

```
quarkus.log.category.docker.level=ERROR
```

But then we lose the ability to distinguish different containers.

I think a better approach would be to create a hierarchy of `docker.<image name>.<label>`, separated by dots. 

Also, IMHO the little whale emoji is pretty, but only adds clutter when included in a logger name. Maybe move it to the log messages themselves?

Reply:
I agree @adutra. The whale emoji is cute but it totally doesn't belong in the actual logger _name_. :joy: So something like this would be preferable here:

```java
        return LoggerFactory.getLogger("docker." + abbreviatedName);
```

Ideally, I think it should be prefixed with `org.testcontainers` though, so that all Testcontainers-based logging can be filtered away easily. `org.testcontainers.docker` would perhaps be the most suitable prefix here, i.e. this:

```java
        return LoggerFactory.getLogger("org.testcontainers.docker." + abbreviatedName);
```